bedrock/events/utils.py

```diff
@@ -33,27 +34,42 @@ def wrapped(*args, **kwargs):
 
 @cache_memoized
 def current_and_future_event_count():
-    return Event.objects.current_and_future().count()
+    try:
+        return Event.objects.current_and_future().count()
+    except DatabaseError:
+        return None
```

Comment:
Returning `None` may expose us to a thundering herd problem here. It may be better to explicitly cache 0 here instead for a short period of time.